lib/libvcc/vcc_symb.c

```diff
@@ -471,6 +471,14 @@ VCC_GlobalSymbol(struct symbol *sym, vcc_type_t type, const char *pfx)
 	VSB_printf(vsb, "%s_", pfx);
 	VCC_PrintCName(vsb, sym->name, NULL);
 	AZ(VSB_finish(vsb));
+	sym->lname = strdup(VSB_data(vsb));
+	if (type == SUB) {
+		VSB_destroy(&vsb);
+		vsb = VSB_new_auto();
+		AN(vsb);
+		VSB_printf(vsb, "sub_%s", sym->lname);
+		AZ(VSB_finish(vsb));
+	}
 	sym->rname = strdup(VSB_data(vsb));
 	AN(sym->rname);
```

Comment:
We are missing an `AN(sym->lname)` here, but we could also replace both `strdup()` calls with the `REPLACE()` macro.

Reply:
lname/rname are `const`, so `REPLACE()` is not ideal. But The `AN()` is missing, thank you